rotkehlchen/serialization/deserialize.py

```diff
@@ -534,6 +535,16 @@ def deserialize_evm_transaction(
         to_address = deserialize_evm_address(data['to']) if is_empty_to_address else None
         value = read_integer(data, 'value', source)
 
+        authorization_list = None
+        if (raw_authorization_list := data.get('authorizationList')) is not None:
+            authorization_list = [
+                EvmTransactionAuthorization(
+                    nonce=read_integer(entry, 'nonce', source),
+                    delegated_address=deserialize_evm_address(entry['address']),
+                )
+                for entry in raw_authorization_list
+            ]
```

Comment:
those can raise deserializationError, I believe it would be best to make it a normal loop and handle the potential errors